scarb/tests/e2e/build.rs

```diff
@@ -225,8 +229,12 @@ fn compile_with_invalid_version() {
 error: failed to parse manifest at `[..]/Scarb.toml`
```

Comment:
Please use `indoc!` macro and its family, it will deal with newlines and indentation.